snap/README.md

```diff
@@ -27,24 +64,34 @@ You can see the current revisions available for your machine's architecture by r
 $ snap info edgexfoundry
 ```
 
-The snap can be installed using `snap install`. To install the snap from the edge channel:
+The snap can be installed using `snap install`. To install the latest stable version:
+
+```bash
+$ sudo snap install edgexfoundry
+```
+
+To install the snap from the edge channel:
 
 ```bash
 $ sudo snap install edgexfoundry --edge
 ```
 
-You can specify install specific releases using the `--channel` option. For example to install the Fuji release of the snap:
+**Note** - in general, installing from the edge channel is only recommended for development purposes. Depending on the state of the current development release, your mileage may vary.
+
+
+You can also specify specific releases using the `--channel` option. For example to install the Ireland (2.0) release of the snap:
 
 ```bash
-$ sudo snap install edgexfoundry --channel=fuji
+$ sudo snap install edgexfoundry --channel=2.0
 
 ```
 
 Lastly, on a system supporting it, the snap may be installed using GNOME (or Ubuntu) Software Center by searching for `edgexfoundry`.
 
-**Note** - the snap has only been tested on Ubuntu Desktop/Server versions 18.04 and 16.04, as well as Ubuntu Core versions 16 and 18.
+**Note** - the snap has only been tested on Ubuntu Desktop/Server LTS releases (16.04 or later), as well as Ubuntu Core (16 or later).
```

Comment:
nit: we changed 16.04 to 18.04 elsewhere, so not sure why we did the opposite here?